packages/bpk-component-carousel/src/BpkCarousel.js

```diff
@@ -0,0 +1,195 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016-2019 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import PropTypes from 'prop-types';
+import React, { Children, Component, createRef } from 'react';
+import { cssModules } from 'bpk-react-utils';
+
+import BpkButton from '../../bpk-component-button';
+import { withButtonAlignment, withRtlSupport } from '../../bpk-component-icon';
+import ArrowLeftIcon from '../../bpk-component-icon/sm/long-arrow-left';
+import ArrowRightIcon from '../../bpk-component-icon/sm/long-arrow-right';
+
+import STYLES from './BpkCarousel.scss';
+
+const AlignedArrowLeft = withButtonAlignment(withRtlSupport(ArrowLeftIcon));
+const AlignedArrowRight = withButtonAlignment(withRtlSupport(ArrowRightIcon));
+
+const getClassName = cssModules(STYLES);
+
+type Props = {
+  className: ?string,
+  wrapperClassName: ?string,
+  itemClassName: ?string,
+  children: Node,
+};
+
+type State = {
+  position: number,
+  prevArrowDisabled: boolean,
+  nextArrowDisabled: boolean,
+};
+
+class BpkCarousel extends Component<Props, State> {
+  wrapperRef: Object;
```

Comment:
```suggestion
  wrapperRef: ?HTMLDivElement;
```

Reply:
This was flows suggestion as that is what element is created and not HTMLDivElement. Changing to your suggestion causes flow to fail.